### ISSUE TITLE
Panel: enhance accessibility for toggleable panels

### ISSUE THREAD

**Issue by EugenFischer** [primefaces/primefaces#9993]
### Description

A toggleable panel should provide additional aria attributes to improve the accessibility.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

The "aria-expanded" attribute should be used to indicate the current state of the panel and the state of controlled elements.
The "aria-controls" attribute should contain the id of panel-body.
Additionaly panel-body should provide "aria-hidden" attribute to be able to hide from screen reader if content is collapsed.

### Describe the solution you would like

Follow example demonstrate the expanded state
```
<div id="form1:panel1"
     class="ui-panel ui-widget ui-widget-content ui-corner-all"
     data-widget="panel"
     role="region"
     aria-labelledby="form1:panel1_header">

    <div id="form1:panel1_header"
         class="ui-panel-titlebar ui-widget-header ui-helper-clearfix ui-corner-all"
         aria-expanded="true"
         aria-controls="form1:panel1_content">
        <span class="ui-panel-title">Header</span>
        <a id="form1:panel1_toggler" href="#" class="ui-panel-titlebar-icon ui-corner-all ui-state-default"
           aria-label="Toggle Panel">
            <span class="ui-icon ui-icon-minusthick"/>
        </a>
    </div>

    <div id="form1:panel1_content"
         class="ui-panel-content ui-widget-content"
         aria-hidden="false">
        <p class="m-0">
            Lorem ipsum dolor sit amet ...
        </p>
    </div>

    <input id="form1:panel1_collapsed" name="form1:panel1_collapsed" type="hidden" autocomplete="off"
           value="false"/>
    <input id="form1:panel1_visible" name="form1:panel1_visible" type="hidden" autocomplete="off"
           value="true"/>
</div>
```

Follow example demonstrate the collapsed state
```
<div id="form1:panel1"
     class="ui-panel ui-widget ui-widget-content ui-corner-all ui-panel-collapsed"
     data-widget="panel"
     role="region"
     aria-labelledby="form1:panel1_header">

    <div id="form1:panel1_header"
         class="ui-panel-titlebar ui-widget-header ui-helper-clearfix ui-corner-all"
         aria-expanded="false"
         aria-controls="form1:panel1_content">
        <span class="ui-panel-title">Header</span>
        <a id="form1:panel1_toggler" href="#" class="ui-panel-titlebar-icon ui-corner-all ui-state-default"
           aria-label="Toggle Panel">
            <span class="ui-icon ui-icon-minusthick"/>
        </a>
    </div>

    <div id="form1:panel1_content"
         class="ui-panel-content ui-widget-content"
         style="display: none;"
         aria-hidden="true">
        <p class="m-0">
            Lorem ipsum dolor sit amet ...
        </p>
    </div>

    <input id="form1:panel1_collapsed" name="form1:panel1_collapsed" type="hidden" autocomplete="off"
           value="true"/>
    <input id="form1:panel1_visible" name="form1:panel1_visible" type="hidden" autocomplete="off"
           value="false"/>
</div>
```

**Comment by melloware** [comment]
Please review my PR.

**Comment by melloware** [comment]
cc @Bukama if you want to review as well.
